src/kibana/plugins/vis_types/vislib/tile_map.js

```diff
@@ -31,6 +32,38 @@ define(function (require) {
           filter.geo_bounding_box[field] = event.bounds;
 
           pushFilter(filter, false, indexPatternName);
+        },
+        mapZoomEnd: function (event) {
```

Comment:
This doesn't check if autoPrecision is turned on, so there's no way to disable it.